scss/main.scss

```diff
@@ -560,6 +560,22 @@ hr {
   }
 }
 
+.about-sponsors {
```

Comment:
can you add a max width of 1440px here to make this section consistent with the other sections on this page? it's a little big on my large monitor: 

![Screen Shot 2022-05-05 at 8 17 35 PM](https://user-images.githubusercontent.com/34359881/167046852-6461e03c-fbda-47f6-a336-cf15230e0c05.png)